tests/test_dummy.py

```diff
@@ -1,7 +0,0 @@
-import pytest
```

Comment:
Unfortunately we probably still need this file for now since we shard the tests in CI so having just one test won't appease `pytest`.